test/python/unittest/API/APITest.py

```diff
@@ -146,12 +146,31 @@ def testData(self):
         # Get a particle species.
         electrons = i.particles["electrons"]
         self.assertIsInstance(electrons, openPMD.ParticleSpecies)
+        pos_y = electrons["position"]["y"]
+        w = electrons["weighting"][openPMD.Record_Component.SCALAR]
 
-        E_x = i.meshes["E"]["x"]
+        self.assertSequenceEqual(pos_y.shape, [270625, ])
+        self.assertSequenceEqual(w.shape, [270625, ])
+        if found_numpy:
+            self.assertEqual(pos_y.dtype, np.float64)
+            self.assertEqual(w.dtype, np.float64)
+        y_data = pos_y.load_chunk([200000, ], [10, ])
+        w_data = w.load_chunk([200000, ], [10, ])
+        series.flush()
+        self.assertSequenceEqual(y_data.shape, [10, ])
+        self.assertSequenceEqual(w_data.shape, [10, ])
+        if found_numpy:
+            self.assertEqual(y_data.dtype, np.float64)
+            self.assertEqual(w_data.dtype, np.float64)
+        # TODO data test
+        # pytest.approx(value)
+        # self.assertSequenceEqual(y_data,
+        #     [-9.60001131e-06, -8.80004967e-06, -8.00007455e-06, ...])
+        # self.assertSequenceEqual(w_data, [1600000., 1600000., ...])
```

Comment:
Is data-transfer for non-mesh ```RecordComponent```s not implemented yet?
If this is commented out because of floating-point comparisons, you could use the same ```np.testing.assert_allclose``` as below.

Reply:
true, I found that later!